release_notes.md

```
@@ -1,20 +1,40 @@
 ### 21Oct24
+
 - Fixed alternative world not have icon make it crash
-- Added toolbar to world when clicked
\ No newline at end of file
+- Added toolbar to world when clicked
+
+### 24Oct24
+
+- Removed Shizuku requirement for default folder access
+- Added settings management system:
+    - Introduced setting.json configuration file
```

Markdownlint

Expected: 2; Actual: 4
Unordered list indentation

(MD007, ul-indent)